04_environment/index.tex

```diff
@@ -1 +1,283 @@
+
+\newpage
+\subsection{Solution Concepts}
+\subsubsection{Dominant Strategy}
+
+In this implementation, there is no dominant strategy. Therefore, if an agent decides to go deer hunting, whereas all the other islands are fishing that agent would not get the best return possible. Hence it is not the best strategy. Whilst if an agent goes fishing and there are a sufficient amount of agents deer hunting, then that agent would be passing on the opportunity of gaining a larger return in deer hunting. Hence, it is not the best strategy either.
+
+\subsubsection{Nash Equilibrium}
+
+In the regular stag hunt game there are two pure Nash equilibria, where both players go for either payoff or risk aversion. In our implementation there are multiple Nash Equilibria; these would be the points where the islands cannot benefit themselves by moving to deers hunting to generate more return. As there is a limit to how many deers that can be hunted thus by moving to deer hunting they are getting a worse return than if they just stayed at fishing. Whilst the islands at the deer hunt are already making a greater return and have no reason to switch to fishing.
+
+\subsubsection{Pareto Optimal Strategy}
+
+At some point all players will be in a position where changing foraging methods will not yield any better income for themselves and in fact also hinder others. This is due to the max daily deer hunt limit which will create a maxim return on the deer hunt. Therefore there will be points where the islands will change as the amount of resources being entered into both methods of foraging is the most efficient, this being where deer returns are maximised. There may also be a point where the user can benefit by switching away from deer hunting, however this will cause the other islands in the deer hunt to have a worse pay off due to a possible drop in tier resulting in worse returns.
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=0.7\textwidth]{04_environment/Images/Pareto Optimal Strategy.PNG}
+    \caption{Pareto Optimal Strategy}
+    \label{fig:Pareto Optimal Strategy}
+\end{figure}
+
+\subsubsection{Social Welfare Maximisation}
+
+It is possible for these islands to find the social welfare maximum in this foraging function with enough time. The islands could identify the exact amount of resources needed, for both fishing and deer hunting, to achieve the best return according to the maximum number of animals. Thus by cooperating who goes where and spends how much they are able to maximize their returns without spending more resources than necessary.
+
+\subsection{Extended Work}
+
+For the future implementations, our additions focus on further exploring the dynamics of our agents and allowing for a more complex decision-making interaction between clients and the environment. Some of these design ideas were later implemented in self-organising multi-agent system and others are expressed for the sake of completeness and future work recommendations.
```

Comment:
our additions "will" focus on..

or maybe 'would'.. but not present tense :)

Reply:
clients ----> islands

Reply:
"Some of these design ideas were later implemented in self-organising multi-agent system "

why do you refer to "self-organising multi-agent system" ?? you can say simply 'coursework', but they way you  are currently referring is not understandable :)